### PR TITLE
#72 Update fargate task

### PR DESCRIPTION
**Pull request by AivarasGluodas**
Extracted notification action as we will need that in a couple of places

### CODE REVIEW

**Review by SSinkus — APPROVED** [review]

File: actions/pull-request-notification/action.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: "pull request notification"
```
Comment (ech018) [inline actions/pull-request-notification/action.yaml:1]:
```suggestion
name: Pull request notification
```

File: actions/pull-request-notification/action.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+description: Github action to setup tse environment infrastructure
```
Comment (ech018) [inline actions/pull-request-notification/action.yaml:2]:
```suggestion
description: Notify pull request author
```

File: actions/update-tse-fargate-instance/action.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: "update tse fargate task"
```
Comment (ech018) [inline actions/update-tse-fargate-instance/action.yaml:1]:
```suggestion
name: ECS service update
```

File: actions/update-tse-fargate-instance/action.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+description: Github action to setup tse environment infrastructure
```
Comment (ech018) [inline actions/update-tse-fargate-instance/action.yaml:2]:
```suggestion
description: Update ecs service cluster and force new deployment
```

File: actions/pull-request-notification/action.yaml (hunk 2 above)
Comment (ech018) [reply actions/pull-request-notification/action.yaml:2]: Or remove field if not mandatory.

File: actions/update-tse-fargate-instance/action.yaml (hunk 4 above)
Comment (ech018) [reply actions/update-tse-fargate-instance/action.yaml:2]: Or remove field if not mandatory.

**Review by ech018 — APPROVED** [review]

**Pull request merged**